.eslintignore

```diff
@@ -1,2 +1,3 @@
 node_modules/*
 coverage/*
+test/*
```

Comment:
I think upgrading ESlint so it can recognise classes would be better here, if the same bug can't be reproduced without the class statement.